app_dart/lib/src/request_handlers/check_for_waiting_pull_requests.dart

```diff
@@ -223,52 +263,36 @@ class CheckForWaitingPullRequests extends ApiRequestHandler<Body> {
     }
     return list;
   }
-}
 
-/// Returns whether all statuses and checks are successful.
-///
-/// Also fills [failures] with the names of any status/check that has failed,
-/// if and only if that status is not `'flutter-build'` or `'luci-engine'`.
-///
-/// The idea is that those statuses are checks based on other pull requests,
-/// not this one. If anything based on this pull request fails, we should
-/// end up removing the label so the user can either fix it or deflake or
-/// what have you.
-bool _checkStatuses(
-  List<Map<String, dynamic>> statuses,
-  List<Map<String, dynamic>> checks,
-  Set<String> failures,
-) {
-  assert(failures != null && failures.isEmpty);
-  // The status checks that are not related to changes in this PR.
-  const Set<String> notInAuthorsControl = <String>{
-    'flutter-build', // flutter repo
-    'luci-engine', // engine repo
-  };
-
-  bool allSuccess = true;
-  for (Map<String, dynamic> status in statuses) {
-    final String name = status['context'];
-    if (status['state'] != 'SUCCESS') {
-      allSuccess = false;
-      if (status['state'] == 'FAILURE' && !notInAuthorsControl.contains(name)) {
-        failures.add(name);
-      }
+  /// Returns whether all statuses are successful.
+  ///
+  /// Also fills [failures] with the names of any status/check that has failed.
+
```

Comment:
nit: remove blank line

also nit: why remove the documentation?

Reply:
Since here we avoid the checks/statuses but use the cirrus API, I just keep those directly related to the cirrus API results. Do you see any part we should keep? I will add them back then.